olxcleaner/loader/xml.py

```diff
@@ -201,7 +201,9 @@ def read_course(edxobj, node, directory, filename, errorstore, htmlfiles, pointe
 
     # The target of a pointer should have no url_name attribute
     if pointer and "url_name" in node.attrib:
-        errorstore.add_error(ExtraURLName(filename, tag=node.tag))
+        _, filename = os.path.split(filename)
+        if not node.attrib['url_name'] == filename.split('.')[0]:
```

Comment:
I recommend using `os.path.splitext` instead of just `split`, in case the filename has multiple periods in it.

Reply:
> I recommend using `os.path.splitext` instead of just `split`, in case the filename has multiple periods in it.

Do you mean as follows? 
```
>>> path, __ = os.path.splitext('abc/def/ghi/filename.with.periods.xml')
('abc/def/ghi/filename.with.periods', '.xml')

>>>os.path.split(path)
('abc/def/ghi', 'filename.with.periods')
```

Reply:
Do split first, then splitext. You already have the filename from 204. (Note: you're overwriting a function argument in 204, which I would tend to avoid!)